### PR TITLE
#28: release 1.1.0, add CD via github actions

### LINKED ISSUE #28: Create release for EasyGecNG-1.1.0
Create a new release for EasyGecNG using the new build system

- [x] Produce .jar
- [x] Create the appropriate zip
- [x] Upload package to github with appropriate release notes
- [x] Update user instructions

### PR DESCRIPTION
**Pull request by qbowers**
- Move `lang.xml` to classpath resources, so it will be included in the release jar and does not need to be manually distributed
- Use symlinks to trick gradle into automatically including top-level documentation in release artifacts
- Add two new github actions to:
  - Create alpha releases on any push to main
  - Create main releases when release tags are created
- Modify java to pick up version from jar manifest
- Modify gradle to save version to jar manifest
- Modify gradle to accept version argument when building artifacts
- Modify CD actions to pass semantic version to gradle

Closes #28

### CODE REVIEW

**Comment by qbowers** [comment]
Oomi's test results:
- need to clarify what java versions are supported
  - add instructions for how to check this
  - add instructions for how to fix this if your java version is too new

**Review by oomip — COMMENTED** [review]
quick few edits

File: README.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ If you want to install a pre-built version of this software (i.e. you are not a
       - `sudo usermod -a -G uucp $USER` on arch-likes (Arch, Manjaro, etc)
     - Log out, log in, run the `group` command, and check that `uucp`/`dialout` appears in the results.
```
[the remark this answers is not in the document]
Comment (qbowers) [reply README.md]: resolved

File: README.md [hunk 2]
```diff
@@ -29,6 +29,9 @@ If you want to install a pre-built version of this software (i.e. you are not a
...
```
[the remark this answers is not in the document]
Comment (qbowers) [reply README.md]: resolved

**Review by oomip — APPROVED** [review]
Looks good to me!

**Pull request merged**